api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -109,4 +109,13 @@ public void teardown()
    */
   public abstract void process(T tuple);
 
+  protected int getCount()
+  {
+    return count;
+  }
+
+  protected void setCount(int count)
```

Comment:
Not needed, instead, provide `protected int incrementCount()`

Reply:
done